docs/src/modules/redux/initRedux.js

```diff
@@ -37,18 +37,3 @@ function create(initialState) {
     compose(applyMiddleware(...middleware), devtools),
   );
 }
-
-export default function initRedux(initialState) {
-  // Make sure to create a new store for every server-side request so that data
-  // isn't shared between connections (which would be bad)
-  if (!process.browser) {
-    return create(initialState);
-  }
-
-  // Reuse store on the client-side
-  if (!global.__INIT_REDUX_STORE__) {
-    global.__INIT_REDUX_STORE__ = create(initialState);
```

Comment:
We init the store on the client side only once anyway since AppWrapper stays mounted.